utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -60,7 +69,40 @@ class UrlImageParser private constructor(
           } else {
             0
           }
-          val rect = Rect(initialDrawableMargin, 0, drawableWidth + initialDrawableMargin, drawableHeight)
+          val rect =
+            Rect(initialDrawableMargin, 0, drawableWidth + initialDrawableMargin, drawableHeight)
+          drawable.bounds = rect
+          urlDrawable.bounds = rect
+          urlDrawable.drawable = drawable
+          htmlContentTextView.text = htmlContentTextView.text
+          htmlContentTextView.invalidate()
+        }
+      }
+    }
+  }
+
+  private inner class SvgBitmapTarget(private val urlDrawable: UrlDrawable) :
```

Comment:
I think we can simplify the new class and consolidate with the old one. Consider:

```kotlin
  private inner class BitmapTarget(urlDrawable: UrlDrawable): CustomImageTarget<Bitmap>(
    urlDrawable, resource -> BitmapDrawable(context.resources, resource)
  )

  private inner class SvgTarget(urlDrawable: UrlDrawable): CustomImageTarget<PictureDrawable>(
    urlDrawable, resource -> PictureDrawable(context.resources, resource.picture)
  )

  private inner class CustomImageTarget<T>(
    private val urlDrawable: UrlDrawable, private val drawableFactory: (T) -> Drawable
  ) : CustomTarget<T>() {
    override fun onLoadCleared(placeholder: Drawable?) {
      // No resources to clear.
    }

    override fun onResourceReady(resource: T, transition: Transition<in T>?) {
      val drawable = drawableFactory(resource)
      htmlContentTextView.post {
        htmlContentTextView.width {
          val drawableHeight = drawable.intrinsicHeight
          val drawableWidth = drawable.intrinsicWidth
          val initialDrawableMargin = if (imageCenterAlign) {
            calculateInitialMargin(it, drawableWidth)
          } else {
            0
          }
          val rect = Rect(initialDrawableMargin, 0, drawableWidth + initialDrawableMargin, drawableHeight)
          drawable.bounds = rect
          urlDrawable.bounds = rect
          urlDrawable.drawable = drawable
          htmlContentTextView.text = htmlContentTextView.text
          htmlContentTextView.invalidate()
        }
      }
    }
  }
```

Reply:
Simplified this. Thanks.